src/sinol_make/commands/export/__init__.py

```diff
@@ -0,0 +1,287 @@
+import os
+import glob
+import shutil
+import tarfile
+import tempfile
+import argparse
+import yaml
+
+from sinol_make import util
+from sinol_make.helpers import package_util, parsers
+from sinol_make.commands.gen import gen_util
+from sinol_make.interfaces.BaseCommand import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Class for "export" command.
+    """
+
+    def get_name(self):
+        return "export"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Create archive for oioioi upload',
+            description='Creates archive ready to upload to oioioi. '
+                        'It is possible to create ocen and dlazaw debian packages for usage in contests '
+                        'with --ocen flag.'
```

Comment:
Why don't we always create the ocen package? Is there a reason not to?

Reply:
This requires an additional tool `dpkg` because the ocen packages are .deb files. And also only OI and OIJ use ocen packages

Reply:
I think that should be a `--dpkg` flag and in the description we should write that it is for the technical team.